.travis.yml

```diff
@@ -137,7 +135,7 @@ before_script:
   - ulimit -c unlimited -S
 
 script:
-  - export MONGODB_URI=`cat /tmp/uri.txt`
+  - export MONGODB_URI=`cat /tmp/uri.txt`$APPEND_URI
```

Comment:
PHPC does this in [`setup_mo.sh`](https://github.com/mongodb/mongo-php-driver/blob/1.6.0alpha3/.travis.scripts/setup_mo.sh#L12) as it writes to `/tmp/uri.txt`. That was based on prior art from some URI modifications it had to do for the `STANDALONE_SSL` environment.

I think one benefit of keeping this in `setup_mo.sh` is that we can ensure the options are appended if `setup_mo.sh` is executed from something other than `.travis.yml`. This may be relevant if we end up standardizing the MO scripts for Evergreen and Travis down the line. I've also run `setup_mo.sh` myself to launch the sever from time to time.

Beyond that, I'd actually love to make a change to MO itself so we can specify extra URI options in the cluster's JSON config file, but that's something further down the road.